libs/data-transforms/src/csv-to-track-segments/guessColumns.ts

```diff
@@ -1,24 +1,44 @@
-const GUESS_COLUMN_NAMES: [string, string[]][] = [
-  ['latitude', ['latitude', 'lat', 'location-lat']],
-  ['longitude', ['longitude', 'lng', 'lon', 'long', 'location-lng']],
-  ['timestamp', ['timestamp', 'time', 'date', 'datetime']],
+export type GuessColum = 'latitude' | 'longitude' | 'timestamp'
```

Comment:
Typo: colum -> column
```suggestion
export type GuessColumn = 'latitude' | 'longitude' | 'timestamp'
```